Utils/release_notes_generator.py

```diff
@@ -97,6 +97,27 @@ def get_new_entity_record(entity_path: str) -> tuple[str, str]:
     return name, description
 
 
+def squash_docker_updates(description: str) -> str:
+    """Squash docker updates to the latest update.
+
+    Args:
+        description: The description containing the docker updates.
+
+    Return: a new description containing only the latest docker update at the bottom.
+    """
+    new_description_lines = []
+    latest_docker_update_line = ''
+    for description_line in description.splitlines():
+        if description_line.startswith('- Updated the Docker image to:'):
```

Comment:
I want to have this a bit more "smart", comparing versions and dockers images, i.e. only filter the same docker images and using comparison choose the higher version. To extract the version you can consult with @JudahSchwartz he has many places he does it in dockerfiles

Reply:
Very rare case but ok. Are we always gurenteed that the last "updated docker.." message is from the latest rn?

Reply:
yes, check out line 389 in this file:

```
    for pack_version, version_release_notes in sorted(pack_versions_dict.items(),
                                                      key=lambda pack_item: Version(pack_item[0])):
```
I also added a unittest for an unsorted entities dict to ensure that if this line changes, it will fail and raise a flag.